src/urbit/app/chess.hoon

```diff
@@ -653,9 +654,15 @@
         ?~  archived-game  ~
         ``[%chess-game !>(u.archived-game)]
       ``[%chess-game !>(game.u.active-game)]
-    ::
-    ::  .^(arch %gy /=chess=/game)
-    ::  collect all the game-id keys
+  ::
+  ::  .^(noun %gx /=chess=/mutuals/noun)
+  ::  .^(json %gx /=chess=/mutuals/json)
+  ::  read mutual pals
+    [%x %mutuals ~]
+      ``[%chess-pals !>((~(mutuals pals bowl) ~.))]
```

Comment:
Should we be targeting mutual contacts, or any contact we've added to `%pals`? "Targets" might be the better wire, since that's the set of everyone we like enough to send them a friend request (whether or not they responded).

Reply:
Either works, we could do both, but I chose mutuals because:

1. Cozier.
2. I wonder if `%pals`-based functionality should lean more towards mutuals than "targets". I've been `target`ed by randos I've never talked to and never see, and I don't want them seeing my mutuals' rumors, for example. `target`ing someone just to access `%pals`-based functionality is kind of spammy and annoying, it shouldn't be encouraged. Something like Facebook gave you very granular permissions about who could see/do what based on whether or not you were Friends with them, and you wouldn't want them extending those permissions to every random person who'd sent you a friend request.
3. To put myself in the randos' position: just because I like someone enough to add them, doesn't mean they like me back :( Are they ignoring my outgoing challenge that's sat there for a few days? Am I being annoying? It's partly why mutuals is cozier.
4. So a game between mutuals *feels* different. Could it work differently? We've talked about the "quick challenge" button, what else might people want to add in future? Example: maybe if our pal breaches in the middle of a long correspondence game, their `%chess` app pokes mutuals for unfinished active games and we can pick up where we left off. And maybe people will want to add more of these privileges to these "friendly games" between people they know and trust. Who knows; a game between mutuals feels like a meaningfully different thing, and something that could be built on top of.

Reply:
I think that's a very solid rebuttal. Let's stick to mutuals.